### PR TITLE
Added Helm chart by Opengov

### PR DESCRIPTION
**Pull request by sakomws**
Thanks for awesome work :)

### CODE REVIEW

**Comment by hjacobs** [comment]
Thanks for your PR!

File: helm-chart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: kube-downscaler
+apiVersion: v1
+version: 0.1.0
+appVersion: 0.5.1
+description:  A Helm chart for kube-downscaler
+keywords:
+- k8s pods scheduler
+- scheduled lifecycle
+home: https://www.opengov.com
```
Comment (hjacobs) [inline helm-chart/Chart.yaml]: Please note that I don't want to use this repo as advertisement space for whatever company/org, so I would prefer to have a more natural "home" link (e.g. URL of this repo). I'm fine with putting your org as contributor/acknowledgement in the README.

**Comment by sakomws** [comment]
Fair enough, we are using it in production :) Removed all company signs, pls check

**Comment by hjacobs** [comment]
Thanks, LGTM, will test it in the next days and consult with my go-to Helm expert :smirk:

**Comment by sakomws** [comment]
awesome, we were so excited seems u had vacation ;)

**Comment by hjacobs** [comment]
@sakomws btw, feel to check out and review/test https://github.com/hjacobs/kube-downscaler/pull/20 - thanks!

**Comment by sakomws** [comment]
sure I will, thx for honor :)

**Pull request merged**